### PR TITLE
Add Danish translation

### PR DESCRIPTION
**Pull request by McGiverGim**
This PR adds Danish language to the Configurator.

Very big thanks to HThuren for the translation!!! A big, big work in only few days!!!

I know this is not a bugfix, but I think we must do an exception and merge this in the current version:
- There can pass a lot of time until the next version is out, so Danish users will not have it's own language until then.
- This new feature is very unlikely that it will break anything.
- Part of the work of the translation will let useless in the next version because some strings will change.

### CODE REVIEW

**Review by haslinghuis — APPROVED** [review]

**Comment by HThuren** [comment]
Hi there, thank you for applying this PR that fast.
I really appreciate your effort. Thanks

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Review by blckmn — APPROVED** [review]

**Pull request merged**
